### PR TITLE
Disable Werror=absolute-value for system libs build

### PR DESCRIPTION
**Pull request by dschuff**
Currently the musl build has a warning where fabs() is called on a long
double. This gets us back to the status quo and unblocks other testing,
but without actually fixing the problem.

### CODE REVIEW

File: tools/system_libs.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def read_symbols(path, exclude=None):
       symbols = filter(lambda symbol: symbol not in exclude, symbols)
     return set(symbols)
 
-  default_opts = ['-Werror']
+  default_opts = ['-Werror', '-Wno-error=absolute-value']
```
Comment (kripken) [inline tools/system_libs.py]: How about

1. Adding that to the list I see exists in `build_libc`, so it just applies to musl (and not other system libs), and
2. Adding it if `shared.Settings.WASM_BACKEND`, as we don't need it in asm.js

? lgtm with that.

Comment (dschuff) [inline tools/system_libs.py]: Done.

**Comment by juj** [comment]
I'd rather we fixed the musl code locally, with an `XXX Emscripten` comment, and not relax these warnings. The reason that `Werror` was added was to explicitly call out issues like these so that we would fix them instead of let them pass. Then in parallel, the issue can be reported to upstream musl so that they'd clean up their codebase as well (if they haven't already).

**Comment by kripken** [comment]
Yeah, I agree, and in the discussion in the other issue we did say fixing upstream is the right thing. For now this was a quick wasm-specific fix to unbreak the wasm waterfall. We should remove it asap.

**Pull request merged**